compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/interop/JInstruction.java

```diff
@@ -39,8 +39,7 @@ public class JInstruction extends BIRNonTerminator {
 
     @Override
     public void accept(BIRVisitor visitor) {
-
-        throw new UnsupportedOperationException();
```

Comment:
I think it is better to keep the `throw new UnsupportedOperationException();` line so that we can find the subclasses which have not overrided this method.